packages/vscode-extension/src/handlers.ts

```diff
@@ -2245,7 +2245,7 @@ export async function migrateTeamsTabAppHandler(): Promise<Result<null, FxError>
   try {
     // Update package.json to use @microsoft/teams-js v2
     await progressBar.next(localize("teamstoolkit.migrateTeamsTabApp.updatingPackageJson"));
-    VsCodeLogInstance.info(localize("teamstoolkit.migrateTeamsTabApp.updatingPackageJson"));
```

Comment:
There's active log refactoring https://github.com/OfficeDev/TeamsFx/pull/9596
So may leave the log APIs as is and only fix others (UI, ...)

Reply:
Good reminder. Let me wait for that PR merged first.

Reply:
Updated.